lib/presentation/pages/tip_information_page.dart

```diff
@@ -49,7 +50,12 @@ class TipInformationPage extends StatelessWidget {
                         const SeparatorAtom(
                           variant: SeparatorVariant.relatedElements,
                         ),
-                        TextAtom(tip.text),
+                        Flexible(
+                          child: TextAtom(
+                            tip.text,
+                            overflow: TextOverflow.clip,
+                          ),
+                        ),
```

Comment:
Do not combine fixes for other issues in one pr!, this slows down the pr and the whole project's progress in one big and cumbersome issue

Reply:
I have reverted this commit for now, will submit another pr when this one is accepted